.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,7 @@
-Before you open a pull request, note that this repository is forked from [here](https://github.com/docascode/sphinx-docfx-yaml/).
-Unless the issue you're trying to solve is unique to this specific repository, 
-please file an issue and/or send changes upstream to the original as well.
```

Comment:
Could you preserve these 3 lines?

Reply:
Yep I'll add it to the `excludes` so it doesn't get overwritten